src/Umbraco.Cms.ManagementApi/Services/Paging/PaginationService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Umbraco.Cms.ManagementApi.Services.Paging;
+
+// TODO: remove this class once EF core is in place with proper skip/take pagination implementation
+// this service is used for converting skip/take to classic pagination with page number and page size.
+// it is a temporary solution that should be removed once EF core is in place, thus we'll live
+// with this code being statically referenced across multiple controllers. the alternative would be
+// an injectable service, but that would require a greater clean-up effort later on.
+internal static class PaginationService
```

Comment:
This is pretty horrid, but EF core will be around to save the day.

It really should have been a `PaginationHelper` but I couldn't bring myself to introduce a `Helpers` namespace in the new management API. I hope we never will need one 😆